cmd/minishift/cmd/start_preflight.go

```diff
@@ -74,7 +74,7 @@ func preflightChecksBeforeStartingHost() {
 			checkOriginRelease,
 			fmt.Sprintf("Checking if requested OpenShift version '%s' is valid", viper.GetString(configCmd.OpenshiftVersion.Name)),
 			configCmd.WarnCheckOpenShiftRelease.Name,
-			"",
+			fmt.Sprintf("Use 'minishift config set %s true' to skip this test", configCmd.SkipCheckOpenShiftRelease.Name),
```

Comment:
@praveenkumar  When the tests fail the message should suggest that user can skip the test on his own risk  because the pre-flight checks are there for a reason and skipping the test might not fix the root cause of the issue.

Reply:
suggestion: ` fmt.Sprintf("If you want to skip the test, run 'minishift config set %s true'", configCmd.SkipCheckOpenShiftRelease.Name)`

Reply:
@LalatenduMohanty I don't think `If you want to skip the test,` needed because this message only come when test failure happen so either user fix it or skip it.

Reply:
> When the tests fail the message should suggest that user can skip the test on his own risk because the pre-flight checks are there for a reason and skipping the test might not fix the root cause of the issue.

@LalatenduMohanty right but my plan is only to have this message for tests which might not be blocker for the VM start.

Reply:
> @LalatenduMohanty right but my plan is only to have this message for tests which might not be blocker for the VM start.

IMO either we should have this for all preflight checks or no message at all. Being selective about the preflight checks will not be optimal from code maintenance point of view.

Reply:
> @LalatenduMohanty I don't think If you want to skip the test, needed because this message only come when test failure happen so either user fix it or skip it.

I intentionally put `if` because I wanted to leave the decision on the user. Because when a preflight test fails user should the root cause itself and should not skip the test.

Reply:
> To skip this test use 'minishift config set %s true' this will give the user the feedback and it the user choice now.

Sounds better

Reply:
`To skip this test use 'minishift config set %s true'` :+1: